java/client/src/main/java/glide/api/models/PubSubMessage.java

```diff
@@ -32,10 +32,10 @@ public PubSubMessage(String message, String channel) {
 
     @Override
     public String toString() {
-        String res = String.format("%s, channel = %s", message, channel);
+        String res = String.format("(%s, channel = %s", message, channel);
```

Comment:
This makes test failures easier to read

Reply:
When we log PubSubMessage, do we use the Supplier<> overload for the logger?

Reply:
We don't log, `junit` does (if a test failed), in format:
```
 org.opentest4j.AssertionFailedError:  expected: <[(message, channel = ...), (message, channel = ...), ...]>, but was: ....
```